src/common/function/src/admin/migrate_region.rs

```diff
@@ -25,12 +25,12 @@ use session::context::QueryContextRef;
 use crate::handlers::ProcedureServiceHandlerRef;
 use crate::helper::cast_u64;
 
-const DEFAULT_TIMEOUT_SECS: u64 = 30;
+const DEFAULT_TIMEOUT_SECS: u64 = 120;
```

Comment:
The timeout may not be enough.
```suggestion
const DEFAULT_TIMEOUT_SECS: u64 = 300;
```